server/controllers/middleware.js

```diff
@@ -0,0 +1,32 @@
+const jwt = require('jsonwebtoken');
+const config = require('../../config/config.js');
+
+const MiddleWare = {
+  Auth(req, res, next) {
+    const token = req.body.token || req.query.token ||
+      req.headers['x-access-token'];
```

Comment:
**Code Quality - Critical**

If the line is more than 80 characters then you should break it and leave a line after. This is to show that it is a multi-line and this is good practice.